Sources/Commands/SwiftPackageTool.swift

```diff
@@ -224,15 +226,15 @@ extension SwiftPackageTool {
         
         func run(_ swiftTool: SwiftTool) throws {
             guard let cwd = localFileSystem.currentWorkingDirectory else {
-                throw InternalError("Could not find the current working directory.")
+                throw MakePackageErrors.currentWorkingDirectoryNotFound
             }
             
             guard let configPath = try swiftTool.getConfigPath() else {
-                throw InternalError("Could not find config path")
+                throw MakePackageErrors.configPathNotFound
```

Comment:
this one should actually stay an InternalError as this is an assertion - this should never be the case